.github/example-run/breakout.ron

```diff
@@ -0,0 +1,3 @@
+(
```

Comment:
I would claim that specifying which example to run in the config would be wise, which would be useful to test diverging paths once we accept input.

Reply:
after thinking about this, I would prefer to keep the example name outside and instead allow file like `breakout-1.ron`, `breakout-2.ron`, ...
having the name inside means that:
* to get it from CI we have to parse the ron in the GitHub action
* once the example started, it is not useful anymore

Reply:
Hmm for now I'm cool with using it as-is. Maybe long term we can write a rust program that orchestrates the tests / parses + runs the ron files. Then the github action could just run the rust program.

Embedding it in the name creates "collision problems". Ex: what if we create a breakout-1.rs example?